.github/workflows/test.yml

```
@@ -1,39 +1,47 @@
       - 'release/**'
       - 'hotfix/**'
-    tags:
-      - '*'
 
 jobs:
+  pr-target-guard:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Vers main — uniquement hotfix/** ou release/**
+        if: github.event_name == 'pull_request' && github.event.pull_request.base.ref == 'main'
+        run: |
```

actionlint (1.7.12)

[error] "github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)